modules/client/src/allowlistVoting/internal/decoding.ts

```diff
@@ -105,29 +108,29 @@ export class AllowlistVotingDecoding {
    * Decoding encoded data or a TX for the execute function
    *
    * @static
-   * @param {(IDecodingTX | BytesLike)} txOrData
+   * @param {(DecodingTXType | BytesLike)} txOrData
    * @return {*}  {{ _proposalId: string }}
    * @memberof AllowlistVotingDecoding
    */
   public static execute(
-    txOrData: IDecodingTX | BytesLike
-  ): { _proposalId: string } {
+    txOrData: DecodingTXType | BytesLike
+  ): { _proposalId: number } {
     const data = AllowlistVotingDecoding.getDecodedData("execute", txOrData);
     return {
-      _proposalId: data._voteId,
+      _proposalId: parseInt(data._voteId.toString()),
```

Comment:
should the proposalId be a number? Addresses and ID's are generally represented by a hex string that can be copied and pasted